core/sys_context.go

```diff
@@ -64,6 +64,32 @@ func (sc *SysContractCallCtx) IsWhitelisted(feeCurrency *common.Address) bool {
 
 // GetGasPriceMinimum retrieves gas price minimum for given fee currency address.
 func (sc *SysContractCallCtx) GetGasPriceMinimum(feeCurrency *common.Address) *big.Int {
+	return sc.gasPriceMinimums.GetGasPriceMinimum(feeCurrency)
+}
+
+type GasPriceMinimums map[common.Address]*big.Int
+
+// GetCurrentGasPriceMinimumMap returns the gas price minimum map for all whitelisted currencies.
+// Note that the CELO currency is keyed by the Zero address.
```

Comment:
I think L72-73 should be moved to for function `GetGasPriceMinimum`

Reply:
well GetGasPriceMinimum returns by currency, but I copied the 'celo currency is keyed by...' since it's also useful there